pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -632,6 +632,15 @@ type MachineConfig struct {
 //  examples:
 //     - value: clusterConfigExample
 type ClusterConfig struct {
+	//   description: |
+	//     Globally unique identifier for this cluster.
+	//     This is generally a UUID.
+	ClusterID string `yaml:"id"`
```

Comment:
this should be probably `,omitempty` as otherwise this would generate empty value after an upgrade (`id: `) ?